Administrator Guide/Thin-Arbiter-Volumes.md

```diff
@@ -0,0 +1,71 @@
+# Thin Arbiter volumes in gluster
+
+Thin Arbiter is a new type of quorum node where granularity of what is
+good and what is bad data is less compared to the traditional arbiter brick.
+In this type of volume, quorum is taken into account at a brick
+level rather than per file basis. If there is even one file that is marked
+bad (i.e. needs healing) on a data brick, that brick is considered bad for
+all files as a whole. So, even different file, if the write fails
+on the other data brick but succeeds on this 'bad' brick we will return
+failure for the write.
+
+
+- [Why Thin Arbiter?](#why-thin-arbiter)
+- [Setting UP Thin Arbiter Volume](#setting-up-thin-arbiter-volume)
+- [How Thin Arbiter works](#how-thin-arbiter-works)
+
+
+# Why Thin Arbiter?
```

Comment:
Please re-order this document

Describe the problem to which the design of this thin arbiter is the answer
Describe how to install/configure it (what defaults, if any)
Then sections for normal user/Ops and developers